lib/PromptSession.ts

```diff
@@ -265,3 +265,20 @@ export class PromptSession {
 		return answers;
 	}
 }
+
+export function chooseTerm() {
```

Comment:
And this one should be marked as `async` really (though await will force it anyway)
Just for style sake, I'd prefer if the extra function was part of the class itself (matching exported class to file name thing), could be static.